packages/import-bundle/test/bundle1.js

```diff
@@ -34,11 +34,6 @@ export function f6ReadGlobal() {
   return globalThis.sneakyChannel;
 }
 
-export function f7WriteGlobal(a) {
-  // this will throw TypeError
-  globalThis.sneakyChannel = a;
-}
-
-export function f8ReadGlobalSubmodule() {
+export function f7ReadGlobalSubmodule() {
```

Comment:
Hm, why remove this test? We should still check that `globalThis` cannot be used as a sneaky channel..

Reply:
The crux of the issue here is that freezing `globalThis` can be optional because freezing it only limits the vulnerability of programs in the same container to one another. With the prior two bundle formats, there is only one compartment so it is necessary and sufficient to freeze a single `globalThis` to protect the application from its third-party dependencies. With this new bundle format, it is neither necessary nor sufficient to freeze the `globalThis`, and whether to freeze `globalThis` should be at the bundled application’s discretion, so that it can be informed by LavaMoat.

I could leave this test in place if I copied and modified the test for the new bundle format, but that seems like an unnecessary maintenance overhead. I could parameterize the test fixture so the constraint could vary. I could alternately go back into compartment mapper and thread a directive to freeze globalThis from `package.json` or a parallel policy JSON file. I had hoped to defer the last point until we had a better grasp of how LavaMoat will integrate.